lpc11uxx/lpc11uxx.svd

```diff
@@ -283,12 +283,12 @@ v7 updates:
 							<enumeratedValues>
 								<name>ENUM</name>
 								<enumeratedValue>
-									<name>MONITOR_MODE_DISABLE</name>
+									<name>DISABLED</name>
```

Comment:
So I assume most changes are renamings like that? Does it really make it more readable when omitting what is disabled?